### DISCUSSION TITLE
IVI references in Wikipedia?

### DISCUSSION THREAD

**Discussion by alejandro5042** [IviFoundation/ivifoundation.github.io#71] [General]
This might be off-topic for this repo, but I stumbled upon a Wikipedia page that talks about IVI. It gets ~10 unique visitors a day:

<https://en.wikipedia.org/wiki/Instrument_Driver#Standards>

Perhaps we should consider how to update that info, especially if one day we have .NET Core -based drivers, @joseph-mueller.

**Comment by joseph-mueller** [comment]
Yes, I agree.  I don't recall much detail about how to make inputs to Wikipedia but I read through and it could benefit from some changes.

The Wikipedia entry for IVI could use some work also.   

All that might follow pretty naturally from cleaning up what we say about ourselves first :)

**Reply by alejandro5042** [reply]
Great! Created #72 to track doing this.
